### PR TITLE
Include deprecation notice in README

### PR DESCRIPTION
**Pull request by jeremiak**
Update README to reflect #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+**Depcrecation Notice** This repo has been depcrecated in favor of the more comprehensive [federalist-docs](https://github.com/18f/federalist-docs) repository where this content [is duplicated](https://federalist-docs.18f.gov/pages/content-guide/)
+
```
Comment (monfresh) [inline README.md:2]: Looks good to me, except for the misspellings of `deprecation` :smile:

Comment (monfresh) [inline README.md:2]: and `deprecated`.

Comment (monfresh) [inline README.md:2]: I would also make the notice a lot bigger so that it stands out.

**Comment by wslack** [comment]
Merging as I only made spelling fixes.

**Pull request merged**